NEMO/admin.py

```diff
@@ -166,6 +166,13 @@ def clean(self):
 		location = cleaned_data.get("_location")
 		phone_number = cleaned_data.get("_phone_number")
 		primary_owner = cleaned_data.get("_primary_owner")
+		description = cleaned_data.get("_description")
+		image = cleaned_data.get("_image")
+
+		if image:
+			# resize image to 500x500 maximum
+			resized_image = resize_image(image, 500)
+			## Don't know how to save the resized image ! 
```

Comment:
@mrampant-nist 
I don't know how to save the resized_image into the original uploaded file.. Can you do it ?